### PR TITLE
added note about elevation

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/2015

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @e0i, for your information

Thank you

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by e0i** [comment]
@dariomws @get-itips @tseward

Peer review needed for this PR. Thanks.

**Review by get-itips — APPROVED** [review]

File: docset/windows/addsadministration/Install-ADServiceAccount.md [hunk 1]
```diff
@@ -237,6 +237,7 @@ A managed service account object is received by the *Identity* parameter.
 * This cmdlet does not work with Active Directory Lightweight Directory Services (AD LDS).
 * This cmdlet does not work with an Active Directory snapshot.
 * This cmdlet does not work with a read-only domain controller.
+* This cmdlet must be run from elevated PowerShell session.
```
Comment (tseward) [inline docset/windows/addsadministration/Install-ADServiceAccount.md:240]: This cmdlet must be run from an elevated PowerShell session.

**Review by tseward — CHANGES_REQUESTED** [review]

File: docset/windows/addsadministration/Install-ADServiceAccount.md (hunk 1 above)
Comment (dariomws) [inline docset/windows/addsadministration/Install-ADServiceAccount.md:240]:
```suggestion
* This cmdlet must be run from an elevated PowerShell session.
```

Agree with @tseward.

**Review by dariomws — APPROVED** [review]

**Comment by dariomws** [comment]
@MaratMussabekov I think we should also add this information to the W2012 R2 article.

**Comment by MaratMussabekov** [comment]
Hello, done, please have a look at #2211 
Thank you!

**Comment by tseward** [comment]
Is this universal? I suspect it isn't when running from a client machine.

**Comment by MaratMussabekov** [comment]
Hello @tseward ,
could you please clarify?
Thank you

**Comment by tseward** [comment]
> Hello @tseward ,
> could you please clarify?
> Thank you

Does this require elevation when running from a client machine?

**Comment by MaratMussabekov** [comment]
Hello,
yes, for remote sessions, including sessions from client computers, elevation is required too.
Thank you

**Pull request merged**